gov2/demotools/question.go

```diff
@@ -0,0 +1,149 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package demotools provides a set of tools that help you write code examples.
+//
+// **Questioner**
+//
+// The questioner is used in interactive examples to ask for input from the user
+// at a command prompt, validate the answer, and re-ask the question if needed.
```

Comment:
re-ask the question -->  ask the question again,

Reply:
done